bouncer/shared/streaming_swaps.ts

```diff
@@ -0,0 +1,83 @@
+import { InternalAsset as Asset, InternalAssets as Assets } from '@chainflip/cli';
+import { randomBytes } from 'crypto';
+import assert from 'assert';
+import { jsonRpc } from './json_rpc';
+import { newAddress, observeBalanceIncrease, observeSwapRequested, SwapRequestType } from './utils';
+import { send } from './send';
+import { observeEvent, observeEvents } from './utils/substrate';
+import { getBalance } from './get_balance';
+
+const NUMBER_OF_CHUNKS = 2;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function brokerApiRpc(method: string, params: any[]): Promise<any> {
+  return jsonRpc(method, params, 'http://127.0.0.1:10997');
+}
+
+async function testMinPriceRefund(inputAsset: Asset, amount: number) {
```

Comment:
Why is this called `testMinPriceRefund`? Looks like no refund happens during the test.

Reply:
I had renamed it already but forgot to push. woops.